### ISSUE TITLE
Release v1.1.1 hotfix

### ISSUE THREAD

**Issue by p-snft** [Inovesa/Inovesa#122]
After #121 is merged, the counter [INOVESA_VERSION_FIX in the v1.1 branch](https://github.com/Inovesa/Inovesa/blob/v1.1/CMakeLists.txt#L28) has to be increased.

Also, the dates in [copyright_notice()](https://github.com/Inovesa/Inovesa/blob/develop/src/HelperFunctions.cpp#L22) should be either updated or removed. (Dates are actually not needed in copyright notes.)

Afterwards, the branch can be merged into `master` to release Inovesa v1.1.1.
